oesukam/andela-sendit
# #162618072 Add create a parcel button to get a quote page

#### What does this PR do?
Add create a parcel button to get a quote page

#### Description of Task to be completed?
To allow users to create a parcel using information provided to the parcel quotation page

#### How should this be manually tested?
- Login
- Navigate to `Get a Quote page`
- Fill the required information to see `Make order button` appear

#### What are the relevant pivotal tracker stories?
`#162618072`
